assessments/NWEA_Map/README.md

```diff
@@ -32,6 +32,15 @@ Required columns:
    - `to`
 </details>
 
+<details>
+<summary> Assessment ID </summary>
+
+The `assessmentIdentifier` used in this repo is "NWEA-Map." Some implementations may wish to spell or format this differently (e.g. "NWEA-MAP"). Double check your implementation and, if necessary, update the following files prior to running:
```

Comment:
@johncmerfeld Does 'update the following files' mean using project composition? I think if so we should state that and maybe link to the PC part of the EM readme

Reply:
Hmm, I don't think I understand project composition well enough to answer confidently, but I think no. All that's needed is to alter some string values in the listed files

Reply:
I guess what I was thinking is that if someone is just installing this bundle, they can't edit the files unless they created a branch of the bundles repo and kept their project on that branch. @susanxiong do you have thoughts here? Curious what Jeffco is doing for this one

Reply:
Oh yeah that's fair. So far I've only been using bundles from a developer's perspective so hadn't thought too much about what more of an "end user" would do. Hm…

Reply:
oop, we don't have it set up to run automatically or anything, so to run MAP files, we have just been pulling down the repo locally and modifying the files as needed before running earthmover.

Reply:
perhaps we make assessment ID a runtime argument

Reply:
Okay gotcha! I think pushing people to use project composition in this situation is ideal since the goal of that feature is to allow people to make edits to the original bundle if something doesn't work for your particular implementation